redirects/16.0.txt

```diff
@@ -41,6 +41,52 @@ applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst a
+applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst
+applications/inventory_and_mrp/inventory/shipping/setup/ups_credentials.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/ups_credentials.rst
+applications/inventory_and_mrp/inventory/shipping/setup/dhl_credentials.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst
+applications/inventory_and_mrp/inventory/management/shipments_deliveries.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
+applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_one_step.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst
+applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
+applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
+applications/inventory_and_mrp/inventory/management/shipments_deliveries/delivery_three_steps.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/delivery_three_steps.rst
+applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst
+applications/inventory_and_mrp/inventory/shipping/operation/invoicing.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/invoicing.rst
+applications/inventory_and_mrp/inventory/shipping/operation/multipack.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/multipack.rst
+applications/inventory_and_mrp/inventory/shipping/operation/labels.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst
+applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/label_type.rst
+applications/inventory_and_mrp/inventory/shipping/operation/dropshipping.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst
+applications/inventory_and_mrp/inventory/shipping/operation/cancel.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/cancel.rst
+applications/inventory_and_mrp/inventory/management/misc/cluster_picking.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/cluster_picking.rst
```

Comment:
Update redirects for the missing toctree files that were removed and add comments

```suggestion
applications/inventory_and_mrp/inventory/management.rst applications/inventory_and_mrp/inventory/product_management.rst                                                                                                       # /inventory_and_mrp/inventory/management -> /inventory_and_mrp/inventory/product_management
applications/inventory_and_mrp/inventory/management/products.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment.rst                                                                        # /inventory_and_mrp/inventory/management/products -> /inventory_and_mrp/inventory/product_management/product_replenishment
applications/inventory_and_mrp/inventory/management/products/strategies.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment/strategies.rst                                                  # /inventory_and_mrp/inventory/management/products/* -> /inventory_and_mrp/inventory/product_management/product_replenishment/*
applications/inventory_and_mrp/inventory/management/products/mto.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment/mto.rst                                                                # /inventory_and_mrp/inventory/management/products/* -> /inventory_and_mrp/inventory/product_management/product_replenishment/*
applications/inventory_and_mrp/inventory/management/products/reordering_rules.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst                                      # /inventory_and_mrp/inventory/management/products/* -> /inventory_and_mrp/inventory/product_management/product_replenishment/*
applications/inventory_and_mrp/inventory/management/products/uom.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom.rst                                                                # /inventory_and_mrp/inventory/management/products/* -> /inventory_and_mrp/inventory/product_management/product_replenishment/*
applications/inventory_and_mrp/inventory/management/products/scrap_inventory.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment/scrap_inventory.rst                                        # /inventory_and_mrp/inventory/management/products/* -> /inventory_and_mrp/inventory/product_management/product_replenishment/*
applications/inventory_and_mrp/inventory/management/lots_serial_numbers.rst applications/inventory_and_mrp/inventory/product_management/product_tracking.rst                                                                  # /inventory_and_mrp/inventory/management/lots_serial_numbers -> /inventory_and_mrp/inventory/product_management/product_tracking
applications/inventory_and_mrp/inventory/management/lots_serial_numbers/differences.rst applications/inventory_and_mrp/inventory/product_management/product_tracking/differences.rst                                          # /inventory_and_mrp/inventory/management/lots_serial_numbers/* -> /inventory_and_mrp/inventory/product_management/product_tracking/*
applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst                                    # /inventory_and_mrp/inventory/management/lots_serial_numbers/* -> /inventory_and_mrp/inventory/product_management/product_tracking/*
applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst applications/inventory_and_mrp/inventory/product_management/product_tracking/lots.rst                                                        # /inventory_and_mrp/inventory/management/lots_serial_numbers/* -> /inventory_and_mrp/inventory/product_management/product_tracking/*
applications/inventory_and_mrp/inventory/management/lots_serial_numbers/expiration_dates.rst applications/inventory_and_mrp/inventory/product_management/product_tracking/expiration_dates.rst                                # /inventory_and_mrp/inventory/management/lots_serial_numbers/* -> /inventory_and_mrp/inventory/product_management/product_tracking/*
applications/inventory_and_mrp/inventory/management/inventory_adjustments.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst                                             # /inventory_and_mrp/inventory/management/inventory_adjustments -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products
applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst                              # /inventory_and_mrp/inventory/management/inventory_adjustments/count_products -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
applications/inventory_and_mrp/inventory/management/warehouses.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst                                                                       # /inventory_and_mrp/inventory/management/warehouses -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management
applications/inventory_and_mrp/inventory/management/warehouses/resupply_warehouses.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/resupply_warehouses.rst                               # /inventory_and_mrp/inventory/management/warehouses/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouse_replenishment_transfer.rst     # /inventory_and_mrp/inventory/management/warehouses/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst                             # /inventory_and_mrp/inventory/management/warehouses/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst                   # /inventory_and_mrp/inventory/management/warehouses/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
applications/inventory_and_mrp/inventory/management/warehouses/use_locations.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst                                           # /inventory_and_mrp/inventory/management/warehouses/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
applications/inventory_and_mrp/inventory/routes/concepts/use_routes.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_routes.rst                                                       # /inventory_and_mrp/inventory/routes/concepts/use_routes -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
applications/inventory_and_mrp/inventory/management/reporting.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation.rst                                                                         # /inventory_and_mrp/inventory/management/reporting -> /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation
applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/inventory_valuation_config.rst                   # /inventory_and_mrp/inventory/management/reporting/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/*
applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/using_inventory_valuation.rst                     # /inventory_and_mrp/inventory/management/reporting/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/*
applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/integrating_landed_costs.rst                       # /inventory_and_mrp/inventory/management/reporting/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/*
applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/cycle_counts.rst                         # /inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
applications/inventory_and_mrp/inventory/management/misc.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse.rst                                                                    # /inventory_and_mrp/inventory/management/misc -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse
applications/inventory_and_mrp/inventory/management/misc/owned_stock.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/owned_stock.rst                                            # /inventory_and_mrp/inventory/management/misc/* -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
applications/inventory_and_mrp/inventory/management/misc/batch_transfers.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/batch_transfers.rst                                    # /inventory_and_mrp/inventory/management/misc/* -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
applications/inventory_and_mrp/inventory/management/misc/wave_transfers.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/wave_transfers.rst                                      # /inventory_and_mrp/inventory/management/misc/* -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
applications/inventory_and_mrp/inventory/management/misc/cluster_picking.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/cluster_picking.rst                                    # /inventory_and_mrp/inventory/management/misc/* -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
applications/inventory_and_mrp/inventory/routes.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/advanced_operations_warehouse.rst                                                        # /inventory_and_mrp/inventory/routes -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/advanced_operations_warehouse
applications/inventory_and_mrp/inventory/routes/concepts.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/advanced_operations_warehouse.rst                                               # /inventory_and_mrp/inventory/routes/concepts -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/advanced_operations_warehouse
applications/inventory_and_mrp/inventory/routes/concepts/cross_dock.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/cross_dock.rst                                              # /inventory_and_mrp/inventory/routes/concepts/cross_dock -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/stock_warehouses.rst                                  # /inventory_and_mrp/inventory/routes/concepts/stock_warehouses -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
applications/inventory_and_mrp/inventory/routes/strategies.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/advanced_operations_warehouse.rst                                             # /inventory_and_mrp/inventory/routes/strategies -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/advanced_operations_warehouse
applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/putaway.rst                                                  # /inventory_and_mrp/inventory/routes/strategies/* -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
applications/inventory_and_mrp/inventory/routes/strategies/removal.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst                                                  # /inventory_and_mrp/inventory/routes/strategies/* -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
applications/inventory_and_mrp/inventory/management/products/usage.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/usage.rst                                                    # /inventory_and_mrp/inventory/management/products/usage -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
applications/inventory_and_mrp/inventory/shipping.rst applications/inventory_and_mrp/inventory/shipping_receiving.rst                                                                                                         # /inventory_and_mrp/inventory/shipping -> /inventory_and_mrp/inventory/shipping_receiving
applications/inventory_and_mrp/inventory/shipping/setup.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration.rst                                                                               # /inventory_and_mrp/inventory/shipping/setup -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration
applications/inventory_and_mrp/inventory/shipping/setup/delivery_method.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/delivery_method.rst                                               # /inventory_and_mrp/inventory/shipping/setup/* -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/*
applications/inventory_and_mrp/inventory/shipping/setup/third_party_shipper.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst                                       # /inventory_and_mrp/inventory/shipping/setup/* -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/*
applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst                                         # /inventory_and_mrp/inventory/shipping/setup/* -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/*
applications/inventory_and_mrp/inventory/shipping/setup/ups_credentials.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/ups_credentials.rst                                               # /inventory_and_mrp/inventory/shipping/setup/* -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/*
applications/inventory_and_mrp/inventory/shipping/setup/dhl_credentials.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst                                               # /inventory_and_mrp/inventory/shipping/setup/* -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/*
applications/inventory_and_mrp/inventory/management/shipments_deliveries.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst                                            # /inventory_and_mrp/inventory/management/shipments_deliveries -> /inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries
applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_one_step.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst           # /inventory_and_mrp/inventory/management/shipments_deliveries/* -> /inventory_and_mrp/inventory/shipping_receiving/daily_operations/*
applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst         # /inventory_and_mrp/inventory/management/shipments_deliveries/* -> /inventory_and_mrp/inventory/shipping_receiving/daily_operations/*
applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst                       # /inventory_and_mrp/inventory/management/shipments_deliveries/* -> /inventory_and_mrp/inventory/shipping_receiving/daily_operations/*
applications/inventory_and_mrp/inventory/management/shipments_deliveries/delivery_three_steps.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/delivery_three_steps.rst                       # /inventory_and_mrp/inventory/management/shipments_deliveries/* -> /inventory_and_mrp/inventory/shipping_receiving/daily_operations/*
applications/inventory_and_mrp/inventory/management/planning.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst                                                 # /inventory_and_mrp/inventory/management/planning -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates
applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst                                 # /inventory_and_mrp/inventory/management/planning/scheduled_dates -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates
applications/inventory_and_mrp/inventory/shipping/operation.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping.rst                                                                  # /inventory_and_mrp/inventory/shipping/operation -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping
applications/inventory_and_mrp/inventory/shipping/operation/invoicing.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/invoicing.rst                                              # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
applications/inventory_and_mrp/inventory/shipping/operation/multipack.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/multipack.rst                                              # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
applications/inventory_and_mrp/inventory/shipping/operation/labels.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst                                                    # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/label_type.rst                                            # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
applications/inventory_and_mrp/inventory/shipping/operation/dropshipping.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst                                        # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
applications/inventory_and_mrp/inventory/shipping/operation/cancel.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/cancel.rst                                                    # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
```